### PR TITLE
DDP-6300 | fix bug for Rare-X

### PR DESCRIPTION
**Pull request by ScalaChamp**
## Context

_Say a few words about the bigger picture beyond this PR. How do your changes improve life for our users? Link directly to the jira ticket (just type "DDP-..." and it'll automatically link) to help your teammates quickly get up to speed on the motivation for these changes._

## Checklist

- [ ] I have labeled the type of changes involved using the `C-*` labels.
- [ ] I have assessed potential risks and labeled using the `R-*` labels.
- [ ] I have considered error handling and alerts, and added `L-*` labels as needed.
- [ ] I have considered security and privacy, and added `I-*` labels as needed
- [ ] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.
- [ ] If applicable, I have discussed the analytics needs at both a platform and study level with Product and instrumented code accordingly.
- [ ] If applicable, my UI/UX changes have passed muster with Product/Design via an over-the-shoulder review, screenshots, etc.

_If unsure or need help with any of the above items, add the `help wanted` label. For items that starts with `If applicable`, if it is not applicable, check it off and add `n/a` in front._

## FUD Score

_Overall, how are you feeling about these changes?_

- [ x] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [ ] They are user-visible in dev as a regular user journey and require no additional instructions.
- [ ] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [ ] I have written automated positive tests
- [ ] I have written automated negative tests
- [ ] I have written zero automated tests but have poked around locally to verify proper functionality
- [ ] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [ ] These changes require no special release procedures--just code!
- [ ] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

### CODE REVIEW

**Review by yufengwng — DISMISSED** [review]

**Comment by ScalaChamp** [comment]
I've copied both of the files from 'DDP-6300_bug_fix' to 'develop' and there are no more merge conflicts.

About the bug: it was a tiny bug, there were two cases where two picklist options were combined into one and I decoupled them into separate picklist options

You may see bug fix description here: 

https://broadinstitute.atlassian.net/browse/DDP-6300

File: study-builder/studies/rarex/i18n/en.json [hunk 1]
```diff
@@ -57,56 +59,56 @@
   "consent_summary_created": "Please sign the Research Consent Form, which will allow RARE-X to obtain biospecimen samples, copies of your medical records, and opt-in to share blood samples and/or other biosamples.",
   "consent_summary_in_progress": "Please finish reviewing and signing the Research Consent Form.",
   "consent_summary_complete": "Thank you for signing the Research Consent Form.",
-  "consent_signature_first_name": "Your complete legal first (given) name *",
+  "consent_signature_first_name": "Your complete legal first (given) name*",
```
Comment (yufengwng) [inline study-builder/studies/rarex/i18n/en.json:62]: There should be a space before the asterisk (`*`).

File: study-builder/studies/rarex/i18n/en.json [hunk 2]
```diff
@@ -255,13 +249,13 @@
...
-  "general_information_patient_country_birthplace": "Country/Territory where the patient was born *",
+  "general_information_patient_country_birthplace": "Country/Territory where the patient was born*",
   "general_information_patient_country_birthplace_req_hint": "Country/Territory cannot be empty",
 
-  "general_information_patient_city_birthplace": "City/Municipality where the patient was born *",
+  "general_information_patient_city_birthplace": "Name of the city/municipality where the patient was born*",
   "general_information_patient_city_birthplace_req_hint": "City/Municipality cannot be empty",
 
-  "general_information_patient_state_birthplace": "State/Province (or other applicable region) where the patient was born *",
+  "general_information_patient_state_birthplace": "State where the patient was born*",
```
Comment (yufengwng) [inline study-builder/studies/rarex/i18n/en.json:258]: This doesn't look right. It should be `State/Province (or other applicable region) ...`

File: study-builder/studies/rarex/i18n/en.json [hunk 3]
```diff
@@ -713,6 +716,5 @@
 
   "prompt_enter_age": "Enter age",
 
-  "cannot_exceed_length_hint": "Cannot exceed 500 characters",
-  "error_date_cannot_be_in_future": "Date cannot be in the future"
+  "error_date_cannot_be_in_future": "Date cannot be in future"
```
Comment (yufengwng) [inline study-builder/studies/rarex/i18n/en.json:719]: Looks like a bad merge...

**Pull request closed without merging**